pkgs/servers/openafs/1.9/default.nix

```diff
@@ -0,0 +1,105 @@
+{ stdenv, buildPackages, fetchurl, which, autoconf, automake, flex
+, yacc , glibc, perl, kerberos, libxslt, docbook_xsl, file
+, docbook_xml_dtd_43, libtool_2
+, withDevdoc ? false, doxygen, dblatex # Extra developer documentation
+, ncurses # Extra ncurses utilities. Needed for debugging and monitoring.
+, tsmbac ? null # Tivoli Storage Manager Backup Client from IBM
+}:
+
+with (import ./srcs.nix { inherit fetchurl; });
+let
+  inherit (stdenv.lib) optional optionalString optionals;
+
+in stdenv.mkDerivation {
+  pname = "openafs";
+  inherit version srcs;
+
+  depsBuildBuild = [ buildPackages.stdenv.cc ];
+  nativeBuildInputs = [ autoconf automake flex libxslt libtool_2 perl
+    which yacc ] ++ optionals withDevdoc [ doxygen dblatex ];
+
+  buildInputs = [ kerberos ncurses ];
+
+  patches = [ ./bosserver.patch ./cross-build.patch ] ++ optional (tsmbac != null) ./tsmbac.patch;
```

Comment:
Any plans for sending the cross build patch upstream?

Reply:
Not in the near future. I started and it turned out that implementing the general case is more complicated but required from upstream.